README.md

```diff
@@ -2,16 +2,20 @@
 
 Source code repo for the pelux.io website
 
-## How to build
+## How to build the web site locally
 
-This website uses [jekyll](https://jekyllrb.com/) to build static HTML. Those are the steps to build it:
+This website uses [jekyll](https://jekyllrb.com/) to build static HTML. You'll need some dependencies to build the site locally. These dependencies are best to pull from upstream since they rely on versions that are known to work together. Install the gems jekyll, bundler, and minima this way;
+
+    sudo gem install jekyll bundler minima
+
+These are the steps to build the web site locally:
 
     git clone <git repo url>
     cd pelux.io
     bundle install
     bundle exec jekyll serve
     
-This will compile the markdown code and start a server at [http://localhost:4000/](http://localhost:4000/) which you can then open in your browser.
```

Comment:
When I ran it 'localhost' didn't resolve for me and instead Jekyll gave me a URL like this;
```
Auto-regeneration: enabled for '/home/jeremiah/GitHub/jeremiah/pelux.io'
Configuration file: /home/jeremiah/GitHub/jeremiah/pelux.io/_config.yml
Server address: http://127.0.0.1:4000/
```
I suspect if one doesn't have 'localhost' in /etc/hosts/ then it won't resolve. In my case I do have localhost in /etc/hosts so I'm not sure why it didn't resolve. I just thought I would re-use what Jekyll prints.